cypress/local-only/tests/integration/dashboard/private-practitioner-dashboard.cy.ts

```diff
@@ -2,6 +2,22 @@ import { loginAsPrivatePractitioner } from '../../../../helpers/authentication/l
 import { practitionerCreatesElectronicCase } from '../../../../helpers/fileAPetition/practitioner-creates-electronic-case';
 
 describe('Private practitioner views dashboard', () => {
+  before(() => {
```

Comment:
Wouldn't we want our cypress tests to be using the new petitioner flow? Why are we using the old flow for these tests?

Reply:
Indeed! Got most of them, working on the rest

Reply:
Donezo